server_fasthttp.go

```diff
@@ -141,11 +143,11 @@ func (u *FastHTTPUpgrader) Upgrade(ctx *fasthttp.RequestCtx, handler FastHTTPHan
 		return u.responseError(ctx, fasthttp.StatusMethodNotAllowed, fmt.Sprintf("%s request method is not GET", badHandshake))
 	}
 
-	if !bytes.Contains(ctx.Request.Header.Peek("Connection"), strUpgrade) {
+	if (!bytes.Contains(ctx.Request.Header.Peek("Connection"), strLowerUpgrade)) && (!bytes.Contains(ctx.Request.Header.Peek("Connection"), strUpperUpgrade)) {
```

Comment:
If you compare lowe case and with a capital case, i prefer to transform the header value to lowercase and compare them

Reply:
and the same in all comparations

Reply:
But i think's it better to use an ascii comparison